### PR TITLE
Tabui

### PR DESCRIPTION
**Pull request by tehseeni**

### CODE REVIEW

**Comment by enochtsang** [comment]
Gotta fix the build and the merge conflicts.

File: ViewLayer/DebugDisplay/Tab/TabUi/TabUi.cpp [hunk 1]
```diff
@@ -35,3 +48,9 @@ QPushButton& TabUi::mpptButton()
 {
     return *ui_->mpptButton;
 }
+void TabUi::showTime()
+{
+    QTime time = QTime::currentTime();
+    QString time_text = time.toString("hh:mm:ss");
```
Comment (bill-luu) [inline ViewLayer/DebugDisplay/Tab/TabUi/TabUi.cpp]: Use `camelCase`

File: src/ViewLayer/DebugDisplay/Tab/TabUi/TabUi.cpp [hunk 2]
```diff
@@ -1,11 +1,40 @@
...
+    ui_->motorButton->setIcon(QIcon(":/Resources/EngineIcon.png"));
+    ui_->motorButton->setIconSize(QSize(35,35));
+    ui_->faultButton->setIcon(QIcon(":/Resources/GearIcon.png"));
+    ui_->faultButton->setIconSize(QSize(35,35));
+    ui_->mpptButton->setIcon(QIcon(":/Resources/MPPTIcon.png"));
+    ui_->mpptButton->setIconSize(QSize(38,38));
+
+    QPixmap pix(":/Resources/SolarCarTeam.png");
+
+    pix = pix.scaled(QSize(25,25));
```
Comment (bill-luu) [inline src/ViewLayer/DebugDisplay/Tab/TabUi/TabUi.cpp]: Try to keep the original proportions on the icon

File: src/ViewLayer/DebugDisplay/Tab/TabUi/TabUi.cpp [hunk 3]
```diff
@@ -1,11 +1,40 @@
...
+    ui_->calgaryTeam->setPixmap(pix);
```
Comment (bill-luu) [inline src/ViewLayer/DebugDisplay/Tab/TabUi/TabUi.cpp]: `calgaryTeam` isn't the best name, lets do `solarCarTeamLogo`

**Pull request merged**